docs/standards.md

```diff
@@ -4,7 +4,29 @@ Since SG2 is a living design library that is home to reusable patterns, it's imp
 
 Any new work contributed into SG2 must meet the following standards:
 
-### PatternLab Organization
+### Workflow standards for contributing to SG2/D8
+
+#### Keep work discrete
+- Encapsulate your work in the smallest discrete chunks possible.
+- Avoid using `rebase` 
+- Avoid branching off of unmerged branches. Doing so can make your code difficult for others to understand and review, which can impede workflow and make working as a team harder; instead, branch off of `develop` whenever possible.
+- Only work on what's in your assigned ticket. If you find that your ticket requires functionality/styling/etc. that is defined in a different ticket, either coordinate with the developer to whom that ticket's assigned, or switch to working on the ticket if unassigned. - only work on the stuff in your assigned JIRA ticket. If the ticket is not assigned to you but you feel you may need to work on it (or need code from it or whatever) reach out to the developer 
```

Comment:
Bold " - only work on the stuff in your assigned JIRA ticket."

Reply:
If you need to work on another ticket:
- talk to the currently assigned developer
- confirm with the lead or PM
- assign the ticket to yourself
- leave a comment on the ticket explaining why you have taken over the assignment
- begin work